pagopa/pn-frontend-e2e
# PN-10053-e-2-e-destinatario-altri-recapiti

### Issue Description
Some PF's tests are failing due to some bad background API integration. In this case the **PN-9318-A40** is failing because it doesn't remove the contacts once the test is finished and this will cause a failure after re-run the test multiple times.

### Changes Made:
- adjusted some feature steps by lowering `waitTime` and fixing correct `webelements`
- adjusted some other feature tests
- added new background API related to `contacts` for legal part
- updated `@After` of  `@recapitiPF` so that revokes all legal contacts at the end of the test in order to clear the data preparation for the next re-runs

### How to Test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_visualizzazioneSezioneAltriRecapitiPF` and verify if the test works fine.
Also a run into AWS CodeBuild is necessary in order to test everything.
